### PR TITLE
Line height is distributed unevenly when lineHeight <= fontSize

### PR DESCRIPTION
**Pull request by nuoyz**
**Description**

When lineHeight is less than the fontSize, Text shrinks the line's bounding box only by removing space from above the text, rather than distributing the space evenly above and below, as is done with extra line height.

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Resolves #???
image
image

### CODE REVIEW

**Comment by nuoyz** [comment]
@rickycodes done

File: ios/Podfile.lock [hunk 1]
```diff
@@ -672,7 +672,7 @@ SPEC CHECKSUMS:
   Branch: 6a281514287f99d707615ac62c2cca69e0213df0
   BVLinearGradient: e3aad03778a456d77928f594a649e96995f1c872
   CocoaAsyncSocket: 065fd1e645c7abab64f7a6a2007a48038fdc6a99
-  DoubleConversion: 831926d9b8bf8166fd87886c4abab286c2422662
+  DoubleConversion: cde416483dac037923206447da6e1454df403714
```
Comment (gantunesr) [inline ios/Podfile.lock:675]: here

File: ios/Podfile.lock [hunk 2]
```diff
@@ -683,7 +683,7 @@ SPEC CHECKSUMS:
   Flipper-RSocket: 127954abe8b162fcaf68d2134d34dc2bd7076154
   FlipperKit: 651f50a42eb95c01b3e89a60996dd6aded529eeb
   Folly: b73c3869541e86821df3c387eb0af5f65addfab4
-  glog: 5337263514dd6f09803962437687240c5dc39aa4
+  glog: 40a13f7840415b9a77023fbcae0f1e6f43192af3
```
Comment (gantunesr) [inline ios/Podfile.lock:686]: here

**Review by gantunesr — CHANGES_REQUESTED** [review]
LGTM. But remove the Podfile changes

File: ios/Podfile.lock (hunk 1 above)
Comment (nuoyz) [reply ios/Podfile.lock:675]: https://github.com/MetaMask/metamask-mobile/commit/c2a294a4214fa94d7deb872a413d1035369969a4#diff-b7d85c35701918f25c9b2c418249f9b66fd9b067676e295a9eff493b73ee1752;  this commit  ios/Podfile.lock file changed

**Review by gantunesr — COMMENTED** [review]
Hey @nuoyz, looks like the tests are failing, can you ran this command to update the snapshots?

`yarn test:unit:update`

**Comment by nuoyz** [comment]
> Hey @nuoyz, looks like the tests are failing, can you ran this command to update the snapshots?
> 
> `yarn test:unit:update`

image

**Review by gantunesr — APPROVED** [review]

**Pull request merged**